src/app/models/book-model.ts

```diff
@@ -0,0 +1,28 @@
+import { bookType } from '../book-type.enum';
+import { BookBorrowData } from './book-borrow-data-model';
+
+export class Book {
+  public id: number;
```

Comment:
In real world example id should always be unique, so some kind of GUID or UUID. For now you can leave it as it is.